applications/asset_tracker_v2/src/cloud/azure_iot_hub_integration.c

```diff
@@ -20,9 +20,20 @@ LOG_MODULE_REGISTER(MODULE, CONFIG_CLOUD_INTEGRATION_LOG_LEVEL);
 #define PROP_BAG_CONTENT_ENCODING_KEY "%24.ce"
 #define PROP_BAG_CONTENT_ENCODING_VALUE "utf-8"
 #define PROP_BAG_MESSAGE_ENCODING_COUNT 2
+
 #define PROP_BAG_BATCH_COUNT 3
 #define PROP_BAG_BATCH_KEY "batch"
 
+#define PROP_BAG_AGPS_COUNT 3
+#define PROP_BAG_AGPS_KEY "agps"
+#define PROP_BAG_AGPS_VALUE "get"
```

Comment:
This define shoulf probably be renamed to `PROP_BAG_AGPS_GET_VALUE` to avoid confusion